### ISSUE TITLE
OverflowError: value too large to convert to int

### ISSUE THREAD

**Issue by nalbat** [yihuang/credis#2]
Hello.

I've encountered overflow error while executing command
conn.execute('incrby', 'foo', 2495602449)

Traceback (most recent call last):
  File "test.py", line 4, in <module>
    conn.execute('incrby', 'foo', 2495602449)
  File "base.pyx", line 259, in credis.base.Connection.execute (credis/base.c:4821)
  File "base.pyx", line 176, in credis.base.Connection.send_command (credis/base.c:3080)
  File "base.pyx", line 244, in credis.base.Connection._pack_command (credis/base.c:4491)
  File "base.pyx", line 234, in credis.base.Connection._pack_command_list (credis/base.c:4341)
  File "base.pyx", line 219, in credis.base.Connection._encode (credis/base.c:4013)
OverflowError: value too large to convert to int

While redis server documentation allows such an operation.
http://redis.io/commands/incrby
"This operation is limited to 64 bit signed integers."

I am also able to execute it through redis-cli:
127.0.0.1:6379> incrby 'foo' 2495602449
(integer) 2495602449

Will you please fix this overflow error.

Thanks in advance. Sorry for my bad English.

**Comment by yihuang** [comment]
```
>>> import credis
>>> credis.Connection()
<credis.base.Connection object at 0xb68b7dac>
>>> conn = _
>>> conn.execute('incrby', 'foo', 2495602449)
2495602449L
>>> conn.execute('incrby', 'foo', 2495602449)
4991204898L
>>> conn.execute('incrby', 'foo', 2495602449)
7486807347L
>>> 
```

I've tried both 32bit and 64bit machines with python2.7, don't have problem.
Please make sure you are using the very recent version of credis, and let me know your python version.

**Comment by nalbat** [comment]
Hello.

This error encountered on both python 2.6 and 2.7 with credis 1.0.3 installed from pip on 64bit platform.

I've now upgraded credis from pip to version 1.0.4. There is no error on python 2.7 any more.

But unfortunately this version doesn't work at all on python 2.6. It rises an error during import.
That's why I can't install credis 1.0.4 to our Centos 6 servers.

Python 2.6.8 (unknown, Nov  7 2012, 14:47:45) 
[GCC 4.1.2 20080704 (Red Hat 4.1.2-52)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import credis
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.6/site-packages/credis/__init__.py", line 1, in <module>
    from .base import Connection, RedisProtocolError, RedisReplyError, ConnectionError, AuthenticationError
ImportError: /usr/lib64/python2.6/site-packages/credis/base.so: undefined symbol: PyLong_AsLongAndOverflow

Python documentation states that PyLong_AsLongAndOverflow is new in version 2.7.
https://docs.python.org/2.7/c-api/long.html

**Comment by yihuang** [comment]
I've fixed compilation problem with python2.6. please try it.

**Comment by nalbat** [comment]
Thank you very much. I've tested the current version from github. That's amazing! It works both for python 2.7 and 2.6.

**Comment by yihuang** [comment]
I've released 1.0.5 for it, thanks for your feedback, i'm closing this issue.
